src/common/AndroidUtils.ts

```diff
@@ -288,7 +290,7 @@ export class AndroidSDKUtils {
         platformAPI: string,
         device: string,
         abi: string
-    ): Promise<boolean> {
+    ): Promise<void> {
```

Comment:
Sharing my learning. I wanted to see if we can declare the type for `reject` but in Typescript it's apparently set to `any`. https://github.com/Microsoft/TypeScript/issues/7588#issuecomment-199079907
That's why it's fine that we pass`Error`. :)